arch/power/slide_hash_power8.c

```diff
@@ -0,0 +1,71 @@
+ /* Copyright (C) 2019 Matheus Castanho <msc@linux.ibm.com>, IBM
+  * For conditions of distribution and use, see copyright notice in zlib.h
+  */
+
+#include <altivec.h>
+#include "../../zbuild.h"
+#include "../../deflate.h"
+
+static inline void slide_hash_power8_loop (deflate_state *s,
+      unsigned n_elems, Pos *table_end) __attribute__((always_inline));
+
+static void slide_hash_power8_loop(
+    deflate_state *s,
```

Comment:
Don't need each variable on a separate line, just break at 120.

Reply:
Perhaps also use `static inline` like our other files.

Reply:
Done. Queued up for next push.